src/lxc/cmd/lxc_user_nic.c

```diff
@@ -1330,5 +1323,5 @@ int main(int argc, char *argv[])
 		host_veth_ifidx);
 	free(newname);
 	free(nicname);
-	exit(EXIT_SUCCESS);
+	_exit(EXIT_SUCCESS);
```

Comment:
Since you're calling `_exit()` and `_exit()` doesn't flush output streams you need to call `fflush(stdout)` before the `_exit()`.

Reply:
@brauner generally, we use _exit() when terminating child process forked by parent process using fork() syscall. I checked your commit("lxc-usernsexec: cleanup and bugfixes"). _exit() call is used all cases. Is it prefer to use _exit() call for all cases? :)